atd-toolbox/data_model/python/sql_templates/fatalities_view.sql

```diff
@@ -0,0 +1,48 @@
+drop table fatalities cascade;
+
+drop view if exists fatalities_view cascade;
+
+create or replace view fatalities_view AS
+select 
+people.id as person_id,
+crashes.id as crash_id,
+crashes.crash_id as cris_crash_id,
+units.id as unit_id,
+CONCAT_WS(' ', people.prsn_first_name, people.prsn_mid_name, people.prsn_last_name) as victim_name,
+TO_CHAR(crashes.crash_timestamp at time zone 'US/Central', 'yyyy') AS year,
+CONCAT_WS(' ',
+  crashes.rpt_block_num,
+  crashes.rpt_street_pfx,
+  crashes.rpt_street_name,
+  '(',
+  crashes.rpt_sec_block_num,
+  crashes.rpt_sec_street_pfx,
+  crashes.rpt_sec_street_name,
+  ')') AS location,
+to_char(
+  public.crashes.crash_timestamp at time zone 'US/Central', 'YYYY-MM-DD'
+  ) as crash_date_ct,
```

Comment:
maybe instead we should just use the crash_timestamp and have the whole crash timestamp be in the Crash Date column like we do on the Crashes page 
![image](https://github.com/user-attachments/assets/6bda6d3b-e8eb-4f55-b8ff-e8e6b73e2ed8)